### DISCUSSION TITLE
videojs-frames.js

### DISCUSSION THREAD

**Discussion by Franz78x** [videojs/video.js#7926] [Q&A]
I started studying the videojs player and am using the extension videojs-frames.js

Sorry for the question, perhaps trivial, but I can't find the method in the documentation.
I would like to start the timecode at a certain value.

In practice my video file is not in base 0 but starts from 10:00:00:00 (for example)
and I would like to add the offset value to the timecode.

Has anyone dealt with the situation? can you kindly help me?

Thanks everyone for the kind replies.

**Comment by iksinski** [comment]
Did you find a way?
